src/Tests/Microsoft.Diagnostics.Monitoring.StartupHook.UnitTests/Exceptions/Eventing/ExceptionsEventSourceTests.cs

```diff
@@ -99,11 +107,28 @@ public void ExceptionsEventSource_WriteException_NotEnabled()
 
             using ExceptionsEventListener listener = new();
 
-            source.ExceptionInstance(9, OperationCancelledExceptionMessage);
+            source.ExceptionInstance(9, OperationCancelledExceptionMessage, Array.Empty<ulong>());
 
             Assert.Empty(listener.Exceptions);
         }
 
+        [Theory]
+        [InlineData(0, 0, 0)]
+        [InlineData(1, 42, 7)]
```

Comment:
Any significance to these values?

Reply:
No, they are arbitrary. Want to test with _something_ but not random numbers.